lib/extensions/require_nested.rb

```diff
@@ -15,6 +15,13 @@ def require_nested(name)
       Object.require_nested name
     end
   end
+
+  def require_nested_all
+    Dir.glob("#{Dir.pwd}/app/models/#{self.name.underscore}/*.rb").each do |full_path|
+      name = full_path.split('/').last.split('.').first
```

Comment:
I think we can use File.basename here.  [Link](https://ruby-doc.org/core-2.2.0/File.html#method-c-basename)

Reply:
great! thanks